### ISSUE TITLE
Why are these functions preserved?

### ISSUE THREAD

**Issue by AlexGuteniev** [microsoft/STL#3926]
I see that other preserved functions have `_CRTIMP2` or similar, but not these.
Should these really be preserved?

https://github.com/microsoft/STL/blob/ed8150e099f6124c50dd4f002cd2ab8c429a81e3/stl/src/winapisupp.cpp#L157-L175

**Comment by frederick-vs-ja** [comment]
I guess they are not actually preserved. They don't appear in the result of `dumpbin /exports`, so perhaps we can remove them now.

**Comment by StephanTLavavej** [comment]
We can't remove things that were ever needed from the import lib, but since these weren't being injected there, it does seem quite plausible that they can be removed, and that the "preserved for ABI" comments were mistakenly added too mechanically.

**Comment by AlexGuteniev** [comment]
The header declarations were removed in #1194 .

Looks like it was my decision to keep these, after failure with removing `_CRTIMP2`,  https://github.com/microsoft/STL/pull/1194#issuecomment-673934547:
> Actually probably need to keep not only `_CRTIMP2`, but every function visible from header

And I wasn't corrected. Probably the PR was scary enough to do more removal.

**Comment by StephanTLavavej** [comment]
Yeah, there was a ton going on in that PR, and I didn't notice that they were never declared as exported. We've gotten better at noticing this as the codebase has gotten simpler (a lot of the weirdness in winapisupp was because it had organically grown so huge).
